reference/platform-app-yaml.md

```diff
@@ -228,7 +228,7 @@ application in MB.
 ### Mounts
 
 The `mounts` is an object whose keys are paths relative to the root of
-the application. It's in the form `volume_id[/subpath]`.
+the application (That is, where the `.platform.app.yaml` file lives). It's in the form `volume_id[/subpath]`.
```

Comment:
Aside, shall we also mention that `shared:files` used below is the only `volume_id` that we support right now?